docs/provider/provider-setup.md

```diff
@@ -7,6 +7,12 @@ title: Setup
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+:::warning
+
+With the launch of `lava-testnet-2`, aspects of our provider setup have changed. TLS Certificates are now a required part of Provider setup.
```

Comment:
Could you briefly specify what are the requirements of the TLS certificate here? And in the `provider-tls` guide more details on that? We should aim for TLS 1.3.

Reply:
I don't know what the requirements of them are... @omerlavanet - do you know?

Reply:
Temporarily adding 1.3 w/ you @omerlavanet consensus. @ranlavanet will be consulted after fork.
Commit [7a79c4de26e051a5ba8127cd38281e9514f1d79d](https://github.com/lavanet/docs/pull/151/commits/7a79c4de26e051a5ba8127cd38281e9514f1d79d)